### PR TITLE
Imperial stockpile - minor adjustments

### PR DESCRIPTION
**Pull request by dbenage-cx**
(Note this is targeted at ImperialStockpile branch)

Seemed easier to create PR for trivial issues in case some pending work conflicts.

### CODE REVIEW

File: Empire/Empire.cpp [hunk 1]
```diff
@@ -323,36 +321,35 @@ namespace {
...
             group_pp_available -= group_drawdown;
-            
-            float stockpile_drawdown = allocation == group_drawdown ? 
-                        0.0f : (allocation - group_drawdown);
-//XXX                        0.0f : (allocation - group_drawdown) / stockpile_conversion_rate;
-            DebugLogger() << "allocation " << group_pp_available;
-            DebugLogger() << "group_drawdown " << group_drawdown;
-            DebugLogger() << "stockpile_drawdown" << stockpile_drawdown;
+
+            float stockpile_drawdown = allocation == group_drawdown ?  0.0f : (allocation - group_drawdown);
```
Comment (geoffthemedio) [inline Empire/Empire.cpp]: floating point comparisons should avoid == and prefer >=

File: Empire/Empire.cpp [hunk 2]
```diff
@@ -260,7 +258,7 @@ namespace {
             std::map<std::set<int>, float>::iterator available_pp_it = available_pp.find(group);
```
Comment (geoffthemedio) [inline Empire/Empire.cpp]: prefer auto

File: Empire/Empire.cpp [hunk 3]
```diff
@@ -122,7 +122,7 @@ namespace {
             ErrorLogger() << "CalculateStockpileContribution() passed null empire.  doing nothing.";
             return 0.0f;
         }
-        float stockpile_transfer_efficiency = std::min(1.0f,empire->GetMeter("METER_IMPERIAL_PP_TRANSFER_EFFICIENCY")->Current());      
+        float stockpile_transfer_efficiency = std::min(1.0f,empire->GetMeter("METER_IMPERIAL_PP_TRANSFER_EFFICIENCY")->Current());
```
Comment (geoffthemedio) [inline Empire/Empire.cpp]: prefer space after `,`

File: Empire/Empire.cpp [hunk 4]
```diff
@@ -323,36 +321,35 @@ namespace {
...
-            
+
             //DebugLogger() << "... leaving " << group_pp_available << " PP available to group";
 
 
             // check for completion
             float block_cost = item_cost * queue_element.blocksize;
-            if (block_cost*(1.0f - queue_element.progress) - queue_element.allocated_pp < EPSILON)
-                queue_element.turns_left_to_next_item = 1;            
+            if (block_cost * (1.0f - queue_element.progress) - queue_element.allocated_pp < EPSILON)
```
Comment (geoffthemedio) [inline Empire/Empire.cpp]: I wouldn't add spaces around `*`

**Comment by geoffthemedio** [comment]
If you want to merge into the https://github.com/freeorion/freeorion/tree/ImperialStockpile branch, you'll need to rebase on it.

File: Empire/Empire.cpp (hunk 4 above)
Comment (dbenage-cx) [reply Empire/Empire.cpp]: I've reverted this change, for future style reference I'm a little confused on the conditions where (and which) arithmetic operators should not be surrounded with spaces.

**Comment by dbenage-cx** [comment]
Rebased, could probably be squashed down to `grooming`

**Comment by geoffthemedio** [comment]
re: squashing: except for the debug -> trace, sure... but not necessary.

**Pull request merged**